src/kOS/UserIO/TelnetSingletonServer.cs

```diff
@@ -539,6 +556,7 @@ private void DoOutThread()
                 ContinuousChecks();
 
                 sb.Remove(0,sb.Length); // clear the whole thing.
+                Console.WriteLine("outThread, before lock section");
```

Comment:
ah crud.  I forgot to include the magic word "eraseme" here so I didn't notice this debug line being left behind.  I'll push a new commit in a moment that has it removed.